modules/govuk_jenkins/templates/jobs/extract_app_performance.yaml.erb

```diff
@@ -18,7 +18,32 @@
     builders:
         - shell: |
             PIPENV_VENV_IN_PROJECT=1 pipenv install --three --skip-lock
-            PIPENV_VENV_IN_PROJECT=1 GRAPHITE_URL='https://graphite.<%= app_domain_internal %>' pipenv run python run_summary.py
-            PIPENV_VENV_IN_PROJECT=1 pipenv run python lint_data.py output/*.csv
+            PIPENV_VENV_IN_PROJECT=1 GRAPHITE_URL='https://graphite.<%= app_domain_internal %>' ./run.sh
     triggers:
         - timed: '<%= @cron_schedule %>'
+
+    wrappers:
+      - ansicolor:
+          colormap: xterm
+      - inject-passwords:
+          global: false
+          mask-password-params: true
+          job-passwords:
+            - name: GOOGLE_PRIVATE_KEY_ID
+              password:
+                '<%= @google_private_key_id %>'
```

Comment:
I'm not sure if you need the `@` here, as it doesn't seem to be used for the `app_domain_internal`.

Reply:
As far as I can tell the instance variables are the idiomatic style for ERB templates in puppet so I'll change `app_domain_internal`.